authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+## Transaction fees and treasuries
+
+In Ethereum, most of the [gas](https://bitfalls.com/2017/12/05/ethereum-gas-and-transaction-fees-explained/) spent on transactions is going to be [burned](https://eips.ethereum.org/EIPS/eip-1559). This creates deflationary pressure on the native currency, ether. The transactions being issued can contain a "tip" amount which acts as incentive for validators to prioritize certain transactions. This way, those using the network directly benefit all stakeholders as burned ether adds value to all ether.
+
+In Polkadot, transaction fees are split between the validators and the treasury, 20% - 80%. There is no burning. This is so that the protocol can throttle its inflation and deflation more effectively. As slashing is inherently an unpredictable event, burning the stake during mass-outage events would cause sudden deflation to no fault of the validators - it was an infrastructure or protocol level failure. In those cases, it's better to have the funds ready to pay for fixing things than to burn the funds and discourage network participation. The ratio of Treasury / Validator distribution can be changed through governance (see previous section).
+
+In Polkadot, blocks have reserved space for crucial transactions - like those of fishermen reporting misbehaviors. This allows fishermen to do their job even if the blocks are full.
+
+Like Ethereum, Polkadot also has tips on transactions for dealing with spikes in block space demand.
+
+## Consensus
+
+Ethereum implements finalization through a system called [Gasper](https://our.status.im/two-point-oh-justification-and-finalization/), a blend of LMD GHOST (Last Message Driven Greediest Heaviest Observed SubTree) and Casper FFG (friendly finality gadget). Finalization is the concept of being so certain a block is correct and cannot be undone that being wrong about this claim would result in the punishment of a large part of the network (a large number of validators). Polkadot uses GRANDPA (GHOST-based Recursive ANcestor Deriving Prefix Agreement) to do the same thing.
+
+Both Ethereum and Polkadot separate block production from finalization. In Ethereum, block production doesn't have a special name. In Polkadot, it's called [BABE](https://research.web3.foundation/en/latest/polkadot/BABE/Babe/) - Blind Assignment for Blockchain Extension. This separation makes sure that block production can continue without finalization, theoretically supporting blockchain growth even with a severely reduced number of validators. Once enough are back online, the finalization process can finalize the blocks starting from the last finalized one.
```

Comment:
Doesnt "theoretically supporting blockchain growth even with a severely reduced number of validators" fall down if finality is not reached for a checkpoint in Ethereum?
Same in next paragraph it may be useful to mention safety issues and lack of finality fallback.

Reply:
It's exactly the same in polkadot afaik - chains will keep growing so probabalistic finality kicks in based on number of primary blocks in a fork. In Ethereum, the number of votes (attestations) for a particular block is used for added weight during fork choice when justified/finalized blocks aren't around to guide the validators.

Reply:
The parachain section needs some work in structure and also information about the availability and validity scheme we have. It should be mentioned that we are using much fewer miners than ETH2.0, and we can do it because of the availability and validity scheme. The auction scheme should e in a separate paragraph. There should be a note on how these design decisions impact security. Nested relay chain should maybe not be mentioned at all since we still have to figure out how to do it. I would also avoid using "legit".

Reply:
Parathread section: 
- forward reference to ICMP is not necessary
- the word slot is used in many places: block slot, parachain slot, this slot (?) , parathread slot and sometime slot is not used. It causes a bit confusion.
- what type of auction is used?
- comparison to mining in Bitcoin and Ethereum
-line 40 seems redundant